projects/components/src/overlay/sheet/sheet-overlay.component.ts

```diff
@@ -46,6 +46,10 @@ export class SheetOverlayComponent {
   public readonly rendererInjector: Injector;
   public visible: boolean = true;
 
+  @HostListener('document:keydown.escape', ['$event']) onKeydownHandler() {
+    this.close();
```

Comment:
please move below constructor and add visibility/type declaration. Also, can we add this to modal too?